runner/ansible/roles/2.2.3.exclude/tasks/main.yml

```diff
@@ -0,0 +1,25 @@
+---
+
+- name: Set Test ID
+  set_fact:
+    test_id: '2.2.3.exclude'
+
+- name: "{{ test_id }}.check"
+  shell: |
+    # Check the pacemaker version IS NOT
+    # If not installed, exit with error
+    VERSION_ID=$(rpm -q --qf "%{VERSION}\n" pacemaker) || exit 2
+    [[ "$VERSION_ID" != "{{ expected[test_id] }}" ]] && exit 0
+    exit 1
+  check_mode: no
+  register: config_updated
+  changed_when: config_updated.rc != 0
```

Comment:
We don't we use the `is version` logic here?
By the way, I don't get why you have started using the `exit 2` here.
`exit 1` doesn't do the job?